src/components/StockSearchBar/index.jsx

```diff
@@ -0,0 +1,61 @@
+import {
+    HStack,
+    InputGroup,
+    InputLeftElement,
+    Input,
+    Box,
+    useColorModeValue,
+    Button,
+    InputRightElement,
+    Container,
+} from '@chakra-ui/react';
+import { GoSearch } from 'react-icons/go';
+import { useState } from 'react';
+
+const StockSearchBar = () => {
```

Comment:
I am not sure if it is an issue with the local build but the search bar is not working

Reply:
I wasn't happy with the search bar. I reverted back to the original code and will recode it.